packages/fxa-settings/src/components/SecondaryEmailInputForm/index.stories.tsx

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { InMemoryCache } from '@apollo/client';
+import { MockedProvider } from '@apollo/client/testing';
+import { SecondaryEmailInputForm, CREATE_SECONDARY_EMAIL_MUTATION } from '.';
+import { MockedCache, MOCK_ACCOUNT } from '../../models/_mocks';
+import { GET_INITIAL_STATE } from '../App';
+
+// every unverified email with a functioning "Resend verification"
+// button must have a mock object created per mutation attempt.
+const createMock = (email: string) => ({
+  request: {
+    query: CREATE_SECONDARY_EMAIL_MUTATION,
+    variables: { input: { email } },
+  },
+  error: new Error('Email Address already added'),
+});
+
+storiesOf('Components|SecondaryEmailInputForm', module)
+  .add('Default empty', () => (
+    <MockedCache>
+      <SecondaryEmailInputForm />
+    </MockedCache>
+  ))
+  .add('No secondary email set, primary email unverified', () => {
+    const primaryEmail = {
```

Comment:
Heh so a couple of things:

* In my branch (https://github.com/mozilla/fxa/compare/issues/6192-secondary-email), I've DRY'd up the mock emails a tad and changed the name of `createMock`. I guess this is more of an FYI, I don't have this up in a PR yet because of the second point -
* "primary email unverified" isn't a valid state to access Settings in. I was going to take it out in that branch I just linked, but now I wonder if I should do that in a follow up so you can pull in my changes there. The state where we need to do something a bit different is an unverified _session_ and not a primary email, but I guess I'm thinking here that we don't need states beyond "Default" and "With Tooltip Error shown" or something similar.